tests/test_pyxis.py

```diff
@@ -405,56 +405,30 @@ def test_get_operator_indices_with_unreleased_filtered_out(self, page):
         assert "4.8" not in [i["ocp_version"] for i in indices]
 
     @patch('freshmaker.pyxis.Pyxis._pagination')
-    def test_get_bundles_per_index_image(self, page):
+    def test_get_latest_bundles(self, page):
+        self.maxDiff = None
```

Comment:
I don't think you need it after you done with debugging.

Reply:
Removed.